daemon-scheduler/pkg/deployment/deployment.go

```diff
@@ -255,7 +255,7 @@ func (d deployment) GetCurrentDeployment(ctx context.Context, environmentName st
 		}
 	}
 
-	return nil, errors.Errorf("There are no in-progress or completed deployments in environment %s", environmentName)
+	return nil, nil
```

Comment:
yep, this is what we do on all get..() methods, return nil when not present and handling it appropriately in the caller.